.travis.yml

```diff
@@ -33,7 +33,7 @@ matrix:
         - ssh default sudo dnf install -y podman
       script:
         - ssh default sudo podman build -t test /vagrant
-        - ssh default sudo podman run --privileged --cgroupns=private test make localunittest
+        - ssh default sudo podman run --privileged --cgroupns=private -v /lib/modules:/lib/modules test make localunittest
```

Comment:
Please add a comment line for this

Reply:
Added and pushed

Reply:
Hmm, wonder if this should be a read-only mount?